code/__DEFINES/subsystems.dm

```diff
@@ -159,6 +160,7 @@
 #define FIRE_PRIORITY_NPC			20
 #define FIRE_PRIORITY_PROCESS		25
 #define FIRE_PRIORITY_THROWING		25
+#define FIRE_PRIORITY_REAGENTS		26
```

Comment:
Keep in mind the lower this is the less time you'll have to work with each tick.

Reply:
The tick is (hopefully) low intensity - at least I coded it with minimal weight in mind. I just put it one above process since process is what it used before. What would you recommend?

Reply:
honestly I'm not too sure, this ought to do, keep in mind that SSprocessing is a background subsystem, so it'll fire after all the non background subsystems. this is really just ahead of throwing, which seems about right